sql-join/big-spenders.sql

```diff
@@ -0,0 +1,8 @@
+
+SELECT "p"."amount",
+ "c"."firstName",
+  "c"."lastName"
+FROM "payments" AS p
+JOIN "customers" AS c ON p."customerId" = c."customerId"
```

Comment:
I've always used the "on" syntax like you've used here, but the "using" syntax is actually simpler.